content/06.methods.md

```diff
@@ -258,7 +222,54 @@ We computed the mean FPKM across all samples per gene and for each multi-mapped
 After collapsing gene identifiers, there were a total of 46,400 unique expressed genes in the poly-A dataset and a total of 53,011 unique expressed genes remaining in the stranded dataset.
 More detail can be found in the [`collapse-rnaseq` analysis module](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/collapse-rnaseq).
 
-##### Immune Profiling/Deconvolution
+##### Gene fusion detection
+
+We set up [Arriba v1.1.0](https://github.com/suhrig/arriba/) and STAR-Fusion 1.5.0 [@doi:10.1101/120295] fusion detection tools using CWL on CAVATICA.
+For both these tools we used aligned BAM and chimeric SAM files from STAR as inputs and GRCh38_gencode_v27 GTF for gene annotation.
```

Comment:
```suggestion
For both these tools, we used aligned BAM and chimeric SAM files from STAR as inputs and GRCh38_gencode_v27 GTF for gene annotation.
```

On your end, confirm this should indeed say STAR and not STAR-fusion (i'm assuming what you have is right since STAR-fusion comes next!)

Reply:
yes, it's correct!